src/BeaniesList.js

```diff
@@ -1,8 +1,12 @@
 import BeanieBaby from './BeanieBaby.js';
 
+//TODO add a load spinner
 export default function BeaniesList({ beanieBabies }) {
   return (
     <div className='beanies'>
+      {
+        beanieBabies.map(baby => <BeanieBaby key={baby.id} {...baby} />)
```

Comment:
i love this spreading in of the baby, make it easy to access what you need in the BeanyBaby component!